cmd/monaco/integrationtest/v2/test-resources/special-character-in-config/project/synthetic-monitor/config.yaml

```diff
@@ -3,6 +3,10 @@ configs:
   config:
     name: Dynatrace Homepage Check
     parameters:
+      request:
+        type: file
+        path: request.json
+        escaped: false
       ua_string: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML,
```

Comment:
Currently, this one is still working because nested templating is supported by default. Should we still change it and move it to "request"?

Reply:
Going to check this out, I think actually this should not work as is, so something is wrong here 🤔

Reply:
I have updated `request` to include the missing `references` section. However, currently the file parameter doesn't enforce this. I will tackle that in a follow up PR.

Reply:
Update: this is now fixed: the file parameter enforces it.